src/test/libslic3r/test_extrusion_entity.cpp

```diff
@@ -27,12 +30,13 @@ ExtrusionPaths random_paths(size_t count=10, size_t length=20, float LO=-50, flo
 }
 
 using namespace Slic3r;
+using namespace Slic3r::Test;
 
 SCENARIO("ExtrusionEntityCollection: Polygon flattening") {
     srand(0xDEADBEEF); // consistent seed for test reproducibility.
 
     // Generate one specific random path set and save it for later comparison
-    ExtrusionPaths nosort_path_set {random_paths()};
+    ExtrusionPaths nosort_path_set{ random_paths() };
```

Comment:
Was this the result of an automatic code formatter?

Reply:
probably